packages/charts/src/core/common/scales/types.ts

```diff
@@ -10,6 +10,16 @@ export const NORMALIZED_DOMAIN = [0, 1];
 /** Signature for domain calculator */
 export type DomainCalculator = (chartSeriesSet: IChartSeries<IAccessors>[], scaleId: string, scale: IScale<any>) => any[];
 
+/** Signature for a specific domain calculator that rounds domain to the closes tick */
+export interface DomainCalculatorWithTicks extends DomainCalculator {
+    __domainWithTicks?: true;
```

Comment:
I didn't realize you could put a default value on an interface.

Reply:
This is not default value, it's either true or it's not there. The type is "true"

Reply:
Ah, cool.